### PR TITLE
Implementing outlier checks

### PR DESCRIPTION
**Pull request by sswadkar**

### CODE REVIEW

File: backend/data_validation/data_val_2022.py [hunk 1]
```diff
@@ -126,3 +127,85 @@ def check_for_missing_shooting_zones(
...
+            )
+
+            auto_points_match_team_data = list(
+                map(
+                    lambda team_datum: (
+                        team_datum[0],
+                        team_datum[1] * Constants.RapidReact.AUTO_LOWER_HUB_POINT_VALUE
+                        + team_datum[2]
+                        * Constants.RapidReact.AUTO_UPPER_HUB_POINT_VALUE
+                        + team_datum[3] * Constants.RapidReact.AUTO_TAXI_POINT_VALUE,
```
Comment (Shom770) [inline backend/data_validation/data_val_2022.py]: The formatting of this is a little off -- we have Line 170 on one line, but multiplying team_datum[2] and the AUTO_UPPER_HUB_POINT_VALUE is on two different lines, we should probably keep them on both one line.

File: backend/data_validation/utils.py [hunk 2]
```diff
@@ -7,10 +8,24 @@
...
+@dataclass
+class Constants:
+    @dataclass
+    class RapidReact:
+        AUTO_TAXI_POINT_VALUE = 2
+        AUTO_UPPER_HUB_POINT_VALUE = 4
+        AUTO_LOWER_HUB_POINT_VALUE = 2
+
+        TELEOP_UPPER_HUB_POINT_VALUE = 2
+        TELEOP_LOWER_HUB_POINT_VALUE = 1
```
Comment (Shom770) [inline backend/data_validation/utils.py]: Perhaps we should move this to a constants file and not make nested classes?

**Review by Shom770 — APPROVED** [review]
LGTM! Just a few notes

File: backend/data_validation/utils.py (hunk 2 above)
Comment (sswadkar) [reply backend/data_validation/utils.py]: implemented

**Pull request merged**